### ISSUE TITLE
No Make File in Grbl Directory

### ISSUE THREAD

**Issue by nickw89509** [McNugget6750/K40-grbl-board#1]
Just compiled your code had to copy the make file into the grbl directory

**Comment by McNugget6750** [comment]
You're right... It's because I use the Arduino uploader to compile and upload the code as provided by grbl master...

**Comment by McNugget6750** [comment]
Please check this out, now it's all there: https://github.com/McNugget6750/grbl
